core/ordering/cosipbft/types/chain_test.go

```diff
@@ -203,48 +203,108 @@ func TestChain_Verify(t *testing.T) {
 	genesis, err := NewGenesis(ro)
 	require.NoError(t, err)
 
-	c := NewChain(makeLink(t, genesis.digest), nil)
+	c := NewChain(makeLink(t, genesis.digest, Digest{}), nil)
 
-	err = c.Verify(genesis, fake.VerifierFactory{})
+	err = c.Verify(genesis, genesis.GetHash(), fake.VerifierFactory{})
```

Comment:
Couldn't we call 'genesis.GetHash() only once and save it in a variable to speed up testing ?

Reply:
`GetHash()` is an attribute, we won't gain much by saving it in a variable.